node/testing/src/node/config.rs

```diff
@@ -9,6 +9,14 @@ pub enum NodeTestingConfig {
     Rust(RustNodeTestingConfig),
 }
 
+#[derive(Serialize, Deserialize, Debug, Clone, Default)]
+pub enum TestPeerId {
+    #[default]
+    Derived,
```

Comment:
Can you add commit for `Derived` for a warning that it might cause issues if scenario interacts with an external program, E.g. replayer, ocaml node, which aren't under the control of the framework. In case of multiple parallel runs, peer ids will be same for the nodes in those runs and external program might disconnect one because of that.

Reply:
@binier You mean, a comment, not commit?